infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -88,6 +88,32 @@ public void syncCompleted() {
     info("Syncing completed", Color.GREEN);
   }
 
+  public void headNoMoreOptimisticWhileSyncing() {
+    info("Execution Client caught up Beacon chain while syncing", Color.YELLOW);
+  }
+
+  public void headTurnedOptimisticWhileSyncing() {
+    info("Execution Client fallen behind Beacon chain while syncing", Color.YELLOW);
+  }
+
+  public void headTurnedOptimisticWhileInSync() {
+    info(
+        "Unable to verify head against Execution Client. Waiting the Execution Client to be back in sync",
+        Color.YELLOW);
```

Comment:
This is a particularly interesting case.  I think it might be worth Loggins as a warning - we have the beacon chain in sync but the EL suddenly can't verify it.  That's likely because we've re-orged to a fork the execution client doesn't have and it will mean we stop producing attestations until it catches up again.

So maybe:
```suggestion
    warn(
        "Unable to execute the current chain head block payload because the Execution Client is syncing. Activating optimistic sync of the beacon chain node.",
        Color.YELLOW);
```